runtime/oti/VMHelpers.hpp

```diff
@@ -2252,7 +2252,6 @@ class VM_VMHelpers
 			}
 		}
 	}
-
```

Comment:
no real changes in this file

Reply:
May as well leave it since the blank line is unnecessary.